### ISSUE TITLE
User management - privileges

### ISSUE THREAD

**Issue by Djedouas** [ProjectCentreLimited/TOMs#14]
We need to create or investigate on existing roles to have users and superusers.

**Comment by Djedouas** [comment]
@traffic-orders you can post here examples of situations where you need to do something that you can't, and have to ask support for help.

**Comment by Djedouas** [comment]
@greid12 asks in #68 

> Hi,
> 
> We have a request to add a particular bay. Could i either have permissions to add new bay styles and timings of restrictions to my account or is it possible to add a loading during certain hours restrictions.
> 
> Thanks

You can add new timings and other values in other lists.

With the WRITE CAN CONFIRM account, you can manage these lists:

![image](https://user-images.githubusercontent.com/34267385/204836981-900c5414-8ddc-4bcc-b976-3ebf58c3f065.png)

Right-click on the list you want to add a value in and select "Open Attribute Table":

![2022-11-30_16-20](https://user-images.githubusercontent.com/34267385/204837373-e4edc0fb-041a-4151-a0f5-04375d051c8d.png)

In the attribute table:

1. Activate edit mode
2. Add a new entry
3. Write the description
4. and the label text
5. When finished deactivate the edit mode, a pop-up shows up to save the changes

![image](https://user-images.githubusercontent.com/34267385/204838081-3222feba-19a3-4ff6-9e4b-4cb2d89df04a.png)

**Comment by Djedouas** [comment]
@greid12 For Bay styling, do you mean the color/width/etc... of a specific bay? This can only be done by us at the moment.

**Comment by greid12** [comment]
![image](https://user-images.githubusercontent.com/116794639/205042494-fe176495-38be-4cfb-b88a-7c529656b966.png)

I get met with this error message when trying to add the timings, do you know what this means?

Update: it seems to be a duplicate, however when going to select the timings it does not appear in the drop down menu.

**Comment by greid12** [comment]
@Djedouas I have now managed to add one but it does not appear as an option

**Comment by Djedouas** [comment]
OK @greid12 well done!

All right I don't see it either in the list, is it "Mon-Sun 10:00pm-6:00am" that you added (I see this is the last one in the database)?

**Comment by greid12** [comment]
@Djedouas yes that time frame and also, Mon-Fri 6:30am-10:30am

**Comment by Djedouas** [comment]
Ok @greid12 I see that it's not as straightforward as I thought. There is 2 more steps to do, and you does not have any access for that. I have to do this.

In my opinion, admin users like you should be able to add or remove time periods easily, do you agree?

If so, I will create an enhancement issue for this.

**Comment by greid12** [comment]
Yeah @Djedouas i agree, happy to be given permissions for this so we are able to add and remove when necessary

**Comment by Djedouas** [comment]
Ok. Not a matter of permission though, we need to create a specific user interface for that.

Typically a button to open a dedicated menu.

**Comment by Djedouas** [comment]
@greid12 you should be able to see the new time periods now, if not, log out and back in.

**Comment by greid12** [comment]
@Djedouas The option is available and you can select but the label displays Mon-Sun 6:30-10:30 instead of Mon-Fri for some reason

**Comment by Djedouas** [comment]
@greid12  that's unexpected, the label is correct for me... maybe it's a label from another restriction? Can you zoom in to be sure?

**Comment by greid12** [comment]
@Djedouas It was just an error on my part, the label is not showing for it however. I have added a new time restriction as well (Mon-Sun 9:30am-12:30pm) if you are able to make that live?

**Comment by Djedouas** [comment]
@greid12  done

**Comment by greid12** [comment]
@Djedouas Hi, i have added a time period of 18:30pm-06:30am is it okay to make this live?

**Comment by Djedouas** [comment]
@greid12 done
